packages/twenty-front/src/modules/settings/security/components/SettingsSecurityOptionsList.tsx

```diff
@@ -49,12 +49,11 @@ export const SettingsSecurityOptionsList = () => {
         Icon={IconLink}
         title="Invite by Link"
         description="Allow the invitation of new users by sharing an invite link."
-        onClick={() =>
-          handleChange(!currentWorkspace?.isPublicInviteLinkEnabled)
+        checked={currentWorkspace.isPublicInviteLinkEnabled}
```

Comment:
I'm encountering a problem with this implementation. Since we need to use inputs other than a checkbox, we should use a slot instead of a prop.

https://www.figma.com/design/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=42308-234697&t=YrBrHCCvLIyihv7X-4

Reply:
Great point. We chose to restrict minimize the API surface with @charlesBochet as all Cards were relying on a Toggle component. We can modify the API have one version for toggles and one for inputs, etc.

Reply:
WDYT of this implementation:
https://github.com/twentyhq/twenty/pull/8378/files#diff-778c43a88bc2aed0767352fe3e86f1d8a49204de88fd23a6bd891127e0a4a98cR183-R196

We keep the label implementation with the flexibility to use custom input in the card.